bson/json_util.py

```diff
@@ -639,6 +639,8 @@ def _parse_canonical_regex(doc):
     if len(regex) != 2:
         raise TypeError('Bad $regularExpression must include only "pattern"'
                         'and "options" components: %s' % (doc,))
+    if not isinstance(regex['options'], str):
+        raise TypeError('Bad $regularExpression options, options must be string, was type %s' % (type(regex['options'])))
```

Comment:
We limit our lines to 79 chars according to pep 8: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Done.